app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionsListViewModel.kt

```diff
@@ -181,13 +177,23 @@ class ConnectionsListViewModel(
         }
     }
 
+    //TODO: Display list of active Consents
+    fun onViewConsentsOptionSelected() {
+        Log.d("some", "click view consents")
+    }
+
+    fun refreshConsents() {
+        getConsents()
+    }
+
     //TODO SET AS PRIVATE AFTER CREATING TEST FOR COROUTINE
     fun processDecryptedConsentsResult(result: List<ConsentData>) {
         this.consents = result.groupBy { it.connectionId ?: "" }
         val newListItems = listItemsValues.apply {
             forEach {
                 val consentsSize = consents[it.connectionId]?.size ?: 0
                 it.consentDescription = if (consentsSize > 0)  {
+                    it.hasConsents = true
```

Comment:
move up this assignment. (`it.hasConsents =  consentsSize > 0`).
try to avoid usage of default values.